lib/mwoffliner.lib.js

```diff
@@ -25,6 +25,7 @@ var os = require( 'os' );
 var crypto = require( 'crypto' );
 var unicodeCutter = require( 'utf8-binary-cutter' );
 var htmlMinifier = require('html-minifier');
+const fetch = require('node-fetch')
```

Comment:
"fetch" is very common and about to be standard javascript (it is already available in all major web browser). node-fetch is a common tool for those who write nodejs code on a day-to-day base.